lib/routes/home/widgets/payments_list/dialog/closed_channel_payment_details.dart

```diff
@@ -36,7 +36,7 @@ class ClosedChannelPaymentDetailsWidget extends StatelessWidget {
                 if (paymentMinutiae.paymentType == sdk.PaymentType.ClosedChannel &&
                     paymentMinutiae.closingTxid != null) ...[
                   TxWidget(
-                    txURL: "$blockExplorer/tx/${paymentMinutiae.closingTxid!}",
+                    txURL: "$blockexplorer/tx/${paymentMinutiae.closingTxid!}",
```

Comment:
I do understand the attempt to use the ServiceInjector for the block explorer url but the format is specific to mempoolspace so perhaps we can abstract a block explorer service that format the final url?
Maybe we can set a service with one method "`formatTransactionUrl(txID)`" ?

Reply:
Thanks, I was not sure where to put it. I created a new helper class `MempoolHelper`, feel free to suggest another name and two helper functions.  I ended up creating two, one getter for the mempoolexplorer and one for the formater as you suggested. I landed on two since we often wish to display more than one transaction and do not need unnecessary future handling. Let me know what you think.

Reply:
I would use one service (resolved by the ServiceInjector) called ChainExplorer. On this service we can add methods such as: 
`transactionUrl(string txID)`